stripe/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -258,11 +272,15 @@ class Stripe internal constructor(
      * Should be called via `Activity#onActivityResult(int, int, Intent)}}` to handle the
      * result of a PaymentIntent automatic confirmation (see [confirmPayment]) or
      * manual confirmation (see [handleNextActionForPayment]})
+     *
+     * TODO(smaskell): figure out if we can get account id from the intent
```

Comment:
@mshafrir-stripe any thoughts here? It looks like the account is added as an extra on the intent in some cases, but it doesn't seem consistent

Reply:
You can skip adding it here. We should be passing it through the Intent.

Reply:
We *should* be but we aren't. Is that going to be a problem? If we don't add it here then doesn't that mean we have to also not add it in confirmPayment? Otherwise you could do something like this

```
val stripe = new Stripe(acct_1)
stripe.confirmPayment(this, params, acct_2)
...
stripe.onPaymentResult(requestCode, intent, callback) // uses acc_1 even though payment was for acct_2
```

Should we avoid adding support here (and other places that use onActivityResult) and follow up on that later?